src/app/components/stars-and-count/stars-and-count.js

```diff
@@ -54,7 +54,7 @@ export default function StarsAndCount({rating, count=0, showNumber=false}) {
         <div className="stars-and-count">
             <Stars stars={rating} />
             {showNumber && (count > 0) && roundedRating(rating)}
-            <span>({count || 'no reviews'})</span>
+            <span>({`${count || 'no'} reviews`})</span>
```

Comment:
This will read a bit strange when there's `1 reviews` Does your fancy new i18n framework have a `pluralize` function?  Otherwise I've used https://www.npmjs.com/package/plur before.